samples/unit-tests/datalabels/contrast/demo.js

```diff
@@ -66,11 +66,78 @@ QUnit.test(
             `After updating from contrast color,
             label should have new color (#12500)`
         );
+
+        chart.series[1].remove(false);
+        chart.update({
+            chart: {
+                type: 'bar',
+                backgroundColor: '#000000'
+            },
+            series: [{
+                data: [100],
+                dataLabels: {
+                    enabled: true,
+                    format: 'Some long long long long long long long long text'
+                }
+            }]
+        }, false);
+        chart.yAxis[0].setExtremes(null, 1000, false, false);
+        chart.redraw();
```

Comment:
Few notes here:
- you don't need to fire `chart.redraw();` - you can set the `redraw` flag to `true` like this: `chart.yAxis[0].setExtremes(null, 1000, true, false);`
- the `redraw` flag is `true` by default. Do you need to set `animation` flag to `false`? It could be simple `chart.yAxis[0].setExtremes(null, 1000);`
- do we need to remove the second series? I know, it looks better, but it's additional unnecessary code and with 2 series the test should work exactly the same. I'm good both (with `series.remove()` or without), it's your call to decide

Reply:
And sorry, I should have added 3 separate comments...

Reply:
I would like to remove that 2nd series, to keep the cohesion of the test. Additional line with `chart.redraw();` deleted :)